processor/src/main/java/ca/bc/gov/app/service/legacy/LegacyIndividualPersistenceService.java

```diff
@@ -110,14 +120,16 @@ public Mono<MessagingWrapper<ForestClientDto>> generateForestClient(
                         )
                     )
                     .withClientIdentification(
-                        null == detailEntity.getIdentificationCode()
-                        ? Objects.toString(
-                            detailEntity.getIdentificationCode(),
-                            ProcessorUtil.splitName(
-                                getUser(message, ApplicationConstant.CREATED_BY)
-                            )[0]
-                        )
-                        : detailEntity.getClientIdentification()
+                        ProcessorUtil
```

Comment:
Based on the data acquired from the database (got one BCSC and one STAFF Submission), it should work properly